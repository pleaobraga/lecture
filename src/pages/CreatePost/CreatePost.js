import React, { Component } from 'react'
import { connect } from 'react-redux'

export class CreatePost extends Component {
    render() {
        return <div>FormPost</div>
    }
}

export default CreatePost