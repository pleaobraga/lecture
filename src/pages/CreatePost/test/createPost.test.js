import React from 'react'
import { shallow } from 'enzyme'
import { CreatePost } from "../CreatePost"

describe('CreatePost', () => {

    let createPost = shallow(<CreatePost/>)

    it('render proprely', () => {
        expect(createPost).toMatchSnapshot()
    })

})