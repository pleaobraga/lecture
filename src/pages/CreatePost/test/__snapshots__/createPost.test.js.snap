// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreatePost render proprely 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CreatePost />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "FormPost",
    },
    "ref": null,
    "rendered": "FormPost",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "FormPost",
      },
      "ref": null,
      "rendered": "FormPost",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
